### PR TITLE
Adding gps-timemachine

### PR DESCRIPTION
**Pull request by trey-stafford**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mfisher87** [comment]
I'm willing to be listed as a maintainer

**Comment by trey-stafford** [comment]
@conda-forge/help-python @conda-forge-admin, please ping team

**Comment by trey-stafford** [comment]
conda-forge/help-python

**Comment by trey-stafford** [comment]
@conda-forge/help-python

File: recipes/gps-timemachine/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: "{{ build_number }}"
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  noarch: python
+
+requirements:
+  host:
+    - python ~=3.10
```
Comment (xhochy) [inline recipes/gps-timemachine/meta.yaml:22]: Is this a Python 3.10 only recipe? Then please use the following
```suggestion
    - python 3.10
```

File: recipes/gps-timemachine/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+{% set build_number = 0 %}
+
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gps_timemachine-{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
```
Comment (xhochy) [inline recipes/gps-timemachine/meta.yaml:13]:
```suggestion
  sha256: 7758810c5417e2f7a80834d422d01f55f05709897bd06df107f19d2f8c896fa7
```

File: recipes/gps-timemachine/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+{% set name = "gps-timemachine" %}
+{% set version = "1.1.5" %}
+{% set sha256 = "7758810c5417e2f7a80834d422d01f55f05709897bd06df107f19d2f8c896fa7" %}
+{% set build_number = 0 %}
```
Comment (xhochy) [inline recipes/gps-timemachine/meta.yaml:4]: Please use as few as possible jinja.
```suggestion
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/gps-timemachine/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+{% set name = "gps-timemachine" %}
+{% set version = "1.1.5" %}
+{% set sha256 = "7758810c5417e2f7a80834d422d01f55f05709897bd06df107f19d2f8c896fa7" %}
```
Comment (ocefpaf) [inline recipes/gps-timemachine/meta.yaml:3]:
```suggestion
```

File: recipes/gps-timemachine/meta.yaml (hunk 2 above)
Comment (ocefpaf) [inline recipes/gps-timemachine/meta.yaml:13]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gps_timemachine-{{ version }}.tar.gz
  sha256: 7758810c5417e2f7a80834d422d01f55f05709897bd06df107f19d2f8c896fa7
```

File: recipes/gps-timemachine/meta.yaml (hunk 3 above)
Comment (ocefpaf) [inline recipes/gps-timemachine/meta.yaml:4]:
```suggestion
```

File: recipes/gps-timemachine/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gps_timemachine-{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+build:
+  number: "{{ build_number }}"
```
Comment (ocefpaf) [inline recipes/gps-timemachine/meta.yaml:16]:
```suggestion
  number: 0
```

File: recipes/gps-timemachine/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipes/gps-timemachine/meta.yaml:22]:
```suggestion
    - python >=3.10
    - setuptools >=66
```

File: recipes/gps-timemachine/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  noarch: python
+
+requirements:
+  host:
+    - python ~=3.10
+  run:
+    - python ~=3.10
```
Comment (ocefpaf) [inline recipes/gps-timemachine/meta.yaml:24]:
```suggestion
    - python >=3.10
```

**Review by ocefpaf — COMMENTED** [review]
Please use grayskull to generate Python recipes. Grayskull make it easier for us to review.

See https://github.com/conda-forge/staged-recipes?tab=readme-ov-file#grayskull---recipe-generator-for-python-packages-on-pypi for more information.

File: recipes/gps-timemachine/meta.yaml (hunk 3 above)
Comment (ocefpaf) [reply recipes/gps-timemachine/meta.yaml:4]: Ha! My browser did not update but Uwe was faster than me.

File: recipes/gps-timemachine/meta.yaml (hunk 1 above)
Comment (mfisher87) [reply recipes/gps-timemachine/meta.yaml:22]: Thanks! It should be marked >=3.10 :)

**Comment by mfisher87** [comment]
:wave: Thanks for the review! Re: minimizing Jinja, is that purely for the review process? As the recipe goes in to maintenance, I find having all the values that need to be considered for a common change grouped together to be very helpful.

File: recipes/gps-timemachine/meta.yaml (hunk 3 above)
Comment (mfisher87) [reply recipes/gps-timemachine/meta.yaml:4]: Thanks Uwe and Filipe!

**Comment by mfisher87** [comment]
@xhochy or @ocefpaf Comments addressed and build passed, thank you again! :bow:

**Review by marcelotrevisani — APPROVED** [review]

**Pull request merged**